shared/api-spec/adequacies-request.json

```diff
@@ -16,10 +16,10 @@
             "description": "An arbitrary ID that is locally unique within this request",
             "type": "integer"
           },
-          "lat": {
+          "latitude": {
             "type": "number"
           },
-          "lng": {
+          "longitude": {
```

Comment:
Can we use lat and lng everywhere? Do you care strongly?

If you do decide to use longitude and latitude, don’t forget to update the “required” field a few lines down.

Reply:
Yep, probably a longer PR though, created https://github.com/bayesimpact/tds/issues/251!